packages/components/hub/WalletDashboardHeader.tsx

```diff
@@ -56,20 +62,8 @@ const WalletDashboardHeaderCard: React.FC<WalletDashboardHeaderProps> = ({
           position: "relative",
         }}
       >
-        <BrandText
-          style={{
-            fontSize: 12,
-          }}
-        >
-          {title}
-        </BrandText>
-        <BrandText
-          style={{
-            fontSize: 16,
-          }}
-        >
-          {data}
-        </BrandText>
+        <BrandText style={[fontRegular12]}>{title}</BrandText>
+        <BrandText style={[fontRegular16]}>{data}</BrandText>
```

Comment:
It's a smaaaall detail, but we can remove the arrays usage is styles (I forgot too sometimes lol)
```suggestion
        <BrandText style={fontRegular12}>{title}</BrandText>
        <BrandText style={fontRegular16}>{data}</BrandText>
```